packages/cli-repl/test/e2e-analytics.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from 'chai';
+import { startTestCluster } from '../../../testing/integration-testing-hooks';
+import { eventually } from './helpers';
+import { TestShell } from './test-shell';
+
+describe('e2e Analytics', () => {
+  const replSetName = 'replicaSet';
+  const [rs0, rs1, rs2, rs3] = startTestCluster(
+    [ '--single', '--replSet', replSetName ],
+    [ '--single', '--replSet', replSetName ],
+    [ '--single', '--replSet', replSetName ],
```

Comment:
Do we need the other secondaries? It’s not a big deal, but it does save some time when running the tests to only spawn the servers that we need

Reply:
Good point, I'll test that... The first one will just become the primary... it might be nice to leave them in to have "real secondaries" since we also have to specify `readPreference=secondary` for the tags to properly work.

Reply:
Sure, works for me :+1: